### PR TITLE
Add Hero Devs support

### PR DESCRIPTION
**Pull request by timmywil**
- Added banner message to index frontmatter (supported by https://github.com/jquery/jquery-wp-content/pull/468)
- Edited support messaging in and near download button
- Improved responsiveness of download button
- Added HeroDevs edits to support page

Ref https://github.com/jquery/jquery-wp-content/issues/462
Ref https://github.com/jquery/jquery.com/pull/245
Closes https://github.com/jquery/jquery.com/issues/243

### CODE REVIEW

File: pages/support.md [hunk 1]
```diff
@@ -27,17 +41,11 @@ We are on [Libera Chat IRC](https://libera.chat/) in the `#jquery` channel where
...
-* [jQuery Accessibility mailing list](https://groups.google.com/group/jquery-a11y) (2008-2016).
-
 ## Commercial support
 
 The following companies offer commercial support services for jQuery.
 
-* **[Bocoup](https://bocoup.com/)** builds complex cross device and cross platform JavaScript software for startups, the Fortune 500, and federally funded education projects. Bocoup offers in-depth JavaScript training on a number of subjects including jQuery. Learn more at <https://bocoup.com/services/training> or contact at <https://bocoup.com/contact>.
+[HeroDevs](https://www.herodevs.com/support/jquery-nes?utm_source=jQuery&utm_medium=link&utm_campaign=eol_support_jQuery) offers security and compatibility support for EOL versions of jQuery through the [OpenJS Ecosystem Sustainability Program](https://openjsf.org/ecosystem-sustainability-program) and is an approved commercial support vendor. You can find out more about their Never Ending Support options [here](https://www.herodevs.com/support/jquery-nes?utm_source=jQuery&utm_medium=link&utm_campaign=eol_support_jQuery).
```
Comment (Krinkle) [inline pages/support.md:48]: Avoid "here" links. I suggest linking "NES options" instead, and remove "here".

File: pages/support.md [hunk 2]
```diff
@@ -4,6 +4,20 @@
 }
 </script>
 
+## Version Support Status
```
Comment (Krinkle) [inline pages/support.md:7]: I don't know what this means or how to parse it. I suggest "Supported versions" instead, without title case.

File: pages/support.md [hunk 3]
```diff
@@ -4,6 +4,20 @@
...
+
+We support only the latest version of jQuery. The 1.x and 2.x branches are no longer supported.
+
+jQuery 3.x is the current version branch of jQuery.
+
+jQuery 4.0 is in beta. Once jQuery 4.0 is released, we will provide security patches and bug fixes on the 3.x branch for a limited time.
+
+#### Still on an old version? Here are your options
```
Comment (Krinkle) [inline pages/support.md:15]: Under H2, is H3 not H4.

For signal-noise reduction and scan-ability, I suggest omitting "Here are your options".

**Comment by timmywil** [comment]
I can do that.

**Comment by timmywil** [comment]
I made some content changes that I think address the comments. Also, I've added some context and general upgrade instructions to the upgrade guide index page, which was previously just a list of links.

File: pages/upgrade-guide.md [hunk 4]
```diff
@@ -3,8 +3,35 @@
...
+
+### [1.9 Upgrade Guide](/upgrade-guide/1.9/)
+
+### Upgrading jQuery
+
+The jQuery Team provides the [jQuery Migrate plugin](https://github.com/jquery/jquery-migrate) to make upgrading jQuery as easy as possible. It is mainly meant as a development tool that generates warning messages in the browser console that can be used to identify and fix compatibility issues. It temporarily restores deprecated features and behaviors so that older code will still run on newer versions of jQuery while the compatibility issues are addressed.
+
+There are two versions of jQuery Migrate: [1.x](https://github.com/jquery/jquery-migrate/tree/1.x-stable) and [3.x](https://github.com/jquery/jquery-migrate) (there is no Migrate 2.x). Only one version should be used at a time, but you may need to use both in succession if upgrading from a jQuery version that predates jQuery 1.9.
+
+For example, if your current jQuery version is 1.4.4, first use jQuery Migrate 1.x to upgrade to jQuery 1.12.4 and then use jQuery Migrate 3.x to upgrade to the latest jQuery (3.7.1, as of this writing). If your current version is 1.9 or later, you only need to use jQuery Migrate 3.x to upgrade to the latest jQuery.
```
Comment (mgol) [inline pages/upgrade-guide.md:22]: Should we clarify what it means?
> use jQuery Migrate 3.x to upgrade to the latest jQuery (3.7.1, as of this writing)

I know there's an example below but it's for specific versions. How about:
```suggestion
For example, if your current jQuery version is 1.4.4, first upgrade to jQuery 1.12.4 with jQuery Migrate 1.x and then upgrade to the latest jQuery with jQuery Migrate 3.x. If your current version is 1.9 or later, you can update straight to the latest jQuery with jQuery Migrate 3.x.
```
?

File: pages/index.html [hunk 5]
```diff
@@ -2,20 +2,24 @@
 	"title": "jQuery",
 	"pageTemplate": "page-fullwidth.php",
 	"customFields": [
-			{ "key": "hide_title", "value": 1 }
+		{ "key": "banner", "value": "jQuery 4.0 is currently in beta. Once released, jQuery 3.x will only receive critical updates. Learn more about our <a href=\"https://jquery.com/support/\">Version Support</a>." },
```
Comment (mgol) [inline pages/index.html:5]: I assume this may change if https://github.com/jquery/jquery-wp-content/pull/468 changes.

**Review by mgol — COMMENTED** [review]
A few comments, plus this depends on where we end up with https://github.com/jquery/jquery-wp-content/pull/468.

A dedicated support page will definitely be helpful!

File: pages/upgrade-guide.md [hunk 6]
```diff
@@ -3,8 +3,35 @@
 	"noHeadingLinks": true
 }</script>
 
-## [3.5 Upgrade Guide](/upgrade-guide/3.5/)
+## jQuery Upgrade Guides
 
-## [3.0 Upgrade Guide](/upgrade-guide/3.0/)
+The jQuery Upgrade Guides can be helpful when you're looking for more details on a breaking change, or you just want to see the full list of breaking changes for each version. There are upgrade guides for jQuery 1.9, 3.0 and 3.5 that list all of the breaking changes that happened in those releases. Most of the breaking changes listed will probably not apply to your code, but these guides add some context and explanation for each change.
```
[the remark this answers is not in the document]
Comment (timmywil) [reply pages/upgrade-guide.md:8]: I had pulled this from the upgrading blog entry.  I've rewritten this whole paragraph to something I think is more suitable for a page about upgrade guides.

File: pages/index.html (hunk 5 above)
Comment (timmywil) [reply pages/index.html]: This has been removed. See https://github.com/jquery/infrastructure-puppet/pull/64

**Review by mgol — APPROVED** [review]
LGTM

**Review by mgol — APPROVED** [review]

**Pull request merged**